src/font/face/freetype.zig

```diff
@@ -392,42 +382,56 @@ pub const Face = struct {
             // SVG glyphs under FreeType, since that requires bundling another
             // dependency to handle rendering the SVG.
             .no_svg = true,
-        });
+        };
+    }
+
+    /// Get a rect that represents the position and size of the loaded glyph.
+    fn getGlyphSize(glyph: freetype.c.FT_GlyphSlot) font.face.RenderOptions.Constraint.GlyphSize {
```

Comment:
This looks a bit ridiculous. The definition of `GlyphSize` should probably be lifted out of these nested scopes if we want to use it here.